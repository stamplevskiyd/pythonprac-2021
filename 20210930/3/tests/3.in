1,0,3,4
0,2,6,6
1,0,3,7
6,1,8,6
11,0,-1,7
-8,-2,2,7
9,0,45,9
3,0,0,9
