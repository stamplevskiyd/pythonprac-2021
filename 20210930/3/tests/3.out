[50, 0, 134, 70]
[56, -4, 274, 122]
[59, 0, 134, 97]
[148, -2, 356, 175]
